tools/istio-iptables/pkg/dependencies/implementation.go

```diff
@@ -151,12 +152,19 @@ func (r *RealDependencies) DetectIptablesVersion(ipV6 bool) (IptablesVersion, er
 		// if so, immediately use it.
 		return nftVer, nil
 	}
+	// not critical, may find another.
+	log.Debugf("did not find (or cannot use) iptables binary, error was %s: %+v", err, nftVer)
```

Comment:
do we want `%v` instead of `%s` for err value?

Reply:
Good call - actually we should probably use `%w`

Reply:
Yeah, %w is probably better